curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/6610c8cfe4cf4d278e35c156.md

```diff
@@ -14,7 +14,7 @@ As expected, your function now returns `undefined` again. Your `call` variable i
 You should not have a `call` declaration.
 
 ```js
-assert.notMatch(__helpers.removeJSComments(code), /const\s*call/);
+assert.notMatch(__helpers.removeJSComments(code), /const\s+call/);
```

Comment:
this one could maybe stay, if the user removes the space between let and variable name but doesn't delete nothing else the test will pass